google/cloud/sql/connector/instance_connection_manager.py

```diff
@@ -258,7 +259,11 @@ def __init__(
 
         async def _set_instance_data() -> None:
             logger.debug("Updating instance data")
-            self._refresh_in_progress = asyncio.locks.Event(loop=self._loop)
+            # set based on python version
+            if sys.version_info[:3] >= (3, 8):
+                self._refresh_in_progress = asyncio.locks.Event()
+            else:
+                self._refresh_in_progress = asyncio.locks.Event(loop=self._loop)
```

Comment:
Same here. Why do we want this to behave different on two different python versions?